.gitpod.yml

```diff
@@ -0,0 +1,7 @@
+# This configuration file was automatically generated by Gitpod.
+# Please adjust to your needs (see https://www.gitpod.io/docs/config-gitpod-file)
+# and commit this file to your remote git repository to share the goodness with others.
+
+tasks:
```

Comment:
I wonder do we really need this? You cannot debug from the cloud because this is a browser extension.

Reply:
we not need this, I will remove.